app/grants/utils.py

```diff
@@ -212,16 +212,6 @@ def get_user_code(user_id, grant, coding_set=block_codes, length=6):
     return ''.join(coding_id)
 
 
-def add_grant_to_active_clrs(grant):
```

Comment:
This is imported into 2 places but not used -> can we remove those as well ?

Reply:
#9199 fixes that